cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting for database...")
```

Comment:
Consider adding a log level to the `self.stdout.write` calls, such as `self.stdout.write(self.style.NOTICE(...))`, to differentiate between informational and success messages.